language/compiler/bytecode-source-map/src/source_map.rs

```diff
@@ -449,6 +453,20 @@ impl<Location: Clone + Eq> SourceMap<Location> {
                 )) })
     }
 
+    pub fn add_const_mapping(
+        &mut self,
+        const_idx: ConstantPoolIndex,
+        name: ConstantName,
+    ) -> Result<()> {
```

Comment:
@tzakian Thoughts on this being a `Result<()>`?

It currently can't fail, so it feels weird having a result here. But, it might be nice to have for API parity?

Reply:
I think not having a result where we don't need one is probably preferable. But not a strong opinion.